src/components/media_manager/src/media_manager_impl.cc

```diff
@@ -160,6 +164,23 @@ void MediaManagerImpl::Init() {
     streamer_[ServiceType::kAudio]->AddListener(
         streamer_listener_[ServiceType::kAudio]);
   }
+
+  if (application_manager_.hmi_capabilities().pcm_stream_capabilities()) {
+    auto pcm_caps =
```

Comment:
```suggestion
    const auto& pcm_caps =
```

Reply:
Added (but not as ref cause its a pointer) in https://github.com/smartdevicelink/sdl_core/pull/3057/commits/c61c05654ee2d4919b03e0e539eb86ba030f3aba